### PR TITLE
Page for the 2023 MFEM community workshop

### PR DESCRIPTION
**Pull request by tzanio**
Screenshot 2023-07-10 at 10 33 50 AM

### CODE REVIEW

File: src/workshop22.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+about the workshop on LLNL's Computing website. Watch the
+[**video playlist**](https://www.youtube.com/playlist?list=PLy9rIbGDXrG2Cexj3ZH8psiMPKh87O0ff) of
+workshop presentations (linked individually below and available on the [videos page](videos.md)),
+and view **contest submissions** in the [gallery](gallery.md).
+
+![MFEM Logo](img/logo-300.png)
+
+### Overview
+
+The MFEM team is happy to announce the second MFEM Community Workshop, which will
```
Comment (kmittal2) [inline src/workshop22.md:16]:
```suggestion
The MFEM team is happy to announce the third MFEM Community Workshop, which will
```

File: src/workshop22.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+take place on October 25, 2022, virtually, using Zoom for videoconferencing.
```
Comment (kmittal2) [inline src/workshop22.md:17]:
```suggestion
take place on October 25, 2022, virtually, using Zoom for video conferencing.
```

File: src/workshop.md [hunk 3]
```diff
@@ -49,25 +42,25 @@ Workshop participants are encouraged to join the
...
-| 2:00-2:20 | **Break** | All |
-| 2:20-2:40 | **MFEM AWS tutorial** ([Instructions](../tutorial/), [video](https://youtu.be/tsr6vza_7DM)) | **Julian Andrej** (LLNL) |
-| 2:40-3:00 | **Wrap-up & Contest Winners** ([PDF](pdf/workshop22/16_Fisher_MFEM_Workshop_Wrap.pdf), [video](https://youtu.be/7uExaQm8Dmk)) | **Aaron Fisher** (LLNL) |
+| 8:00-8:20 | **Welcome & Overview** | **TBD** (LLNL) |
+| 8:20-8:40 | **The State of MFEM** | **Tzanio Kolev** (LLNL) |
+| 8:40-9:00 | **Recent Developments** | **Veselin Dobrev** (LLNL) |
+| 8:00-9:20 | **Break** | All |
```
Comment (kmittal2) [inline src/workshop.md:54]:
```suggestion
| 9:00-9:20 | **Break** | All |
```

File: src/workshop.md [hunk 4]
```diff
@@ -49,25 +42,25 @@ Workshop participants are encouraged to join the
...
+| 9:20-10:20 | **Talks, Session I** | (20 mins each) |
+| 10:20-10:40 | **Break & Group Photo**| Download a virtual background below |
```
Comment (hauten) [inline src/workshop.md:56]:
```suggestion
| 10:20-10:40 | **Break & Group Photo** | Download a virtual background below |
```

**Review by kmittal2 — APPROVED** [review]
Other than some minor comments I left, this looks good to me.

**Review by hauten — APPROVED** [review]
Like Ketan, I made just a couple of small changes

File: src/workshop.md [hunk 5]
```diff
@@ -23,24 +16,24 @@ directions for the project.
...
 - Contributed talks from application developers utilizing MFEM
 - Roadmap discussion for future development
 - Technical discussions in breakout rooms for Electromagnetics, Fluids, and
   Structural Mechanics applications
 
-See also the agenda for the previous [2021](../workshop21) MFEM workshop.
--->
+See also the agenda for the previous [2022](../workshop22), [2021](../workshop21) MFEM workshops.
 
 ### Meeting format
```
Comment (kmittal2) [inline src/workshop.md:38]: Perhaps we could rename this section `Agenda` and move the note about slack to the `Meeting format` section above.

**Comment by tzanio** [comment]
I updated the workshop date and addressed the @kmittal2's and @hauten's comments (thank you!)

@pazner, @homijan, @psocratis and @acfisher -- do you have any additional suggestions?

**Review by psocratis — APPROVED** [review]

**Review by pazner — APPROVED** [review]

**Review by acfisher — APPROVED** [review]

**Comment by tzanio** [comment]
Thanks for all the help and the reviews @kmittal2, @pazner, @acfisher, @psocratis, @hauten and @homijan 🙏 

This will be another awesome workshop 🚀

**Pull request merged**